### PR TITLE
Added commit method

### PR DESCRIPTION
**Pull request by 719Ben**
Without the commit method nothing is saved to the database. Need to add the commit() method in order to be a better example.

### CODE REVIEW

**Comment by 719Ben** [comment]
While it may not be needed for a simple SELECT statement, there should be an example for code that need the commit() method.

**Comment by methane** [comment]
Is that example runs correctly?
Connection object has `.commit()` method.  But cursor may not have it.

**Comment by 719Ben** [comment]
You are correct, it should `.commit()` on the connection object. I have edited the code reflect that.

File: example.py [hunk 1]
```diff
@@ -17,5 +18,6 @@
 for row in cur:
    print(row)
 
-cur.close()
+conn.commit()
+curr.close()
```
Comment (methane) [inline example.py]: s/curr/cur/

**Comment by methane** [comment]
Inserting user in example is bad idea.
We should have more friendly docs and examples.

**Pull request closed without merging**